### ISSUE TITLE
Error with print in IE11

### ISSUE THREAD

**Issue by arnaud-morvan** [camptocamp/ngeo#3020]
Reproduced on the demo : https://geomapfish-demo.camptocamp.net/2.2

Steps to reproduce the issue:
* Run IE11 and open the geomapfish 2.2 demo
* Open the print panel

```
Error: Invalid argument
```

* Click on PDF button

```
TypeError: Object doesn't support this action
```

Expected result: a map print PDF file.

The issue concern the following environment:

* IE11 on Windows 7

Note that it is not possible to load page in debug mode because IE11 does not support ES6.
Note that the ngeo print example does work anymore.
So it's really hard to understand what going wrong.

**Comment by ger-benjamin** [comment]
Same errors on W10 - IE 11.
And on 2.1, so I can see the sources (ES5). Errors:

````
CSS3111: @font-face encountered unknown error.
gmf-icons.eot

Error: Invalid argument.
   at interpolateFnWatchAction (https://geomapfish-demo.camptocamp.net/2.1/wsgi/node_modules/angular/angular.js:9897:17)
   at interpolateFnWatcher (https://geomapfish-demo.camptocamp.net/2.1/wsgi/node_modules/angular/angular.js:12605:17)
   at watchGroupAction (https://geomapfish-demo.camptocamp.net/2.1/wsgi/node_modules/angular/angular.js:17420:13)
   at Scope.prototype.$digest (https://geomapfish-demo.camptocamp.net/2.1/wsgi/node_modules/angular/angular.js:17755:23)
   at Scope.prototype.$apply (https://geomapfish-demo.camptocamp.net/2.1/wsgi/node_modules/angular/angular.js:18021:13)
   at done (https://geomapfish-demo.camptocamp.net/2.1/wsgi/node_modules/angular/angular.js:12002:36)
   at completeRequest (https://geomapfish-demo.camptocamp.net/2.1/wsgi/node_modules/angular/angular.js:12211:7)
   at requestLoaded (https://geomapfish-demo.camptocamp.net/2.1/wsgi/node_modules/angular/angular.js:12139:9)
````

**Comment by arnaud-morvan** [comment]
Thanks @ger-benjamin, I will try to update aeroportdelyon_sig soon.

**Comment by ger-benjamin** [comment]
@arnaud-morvan Be aware that I've merged my change into 2.1
The fix is not already available in 2.2 (it will be in the next merge 2.1->2.2)

**Comment by arnaud-morvan** [comment]
I will probably do a cherry-pick in a fork/branch because I do not want to break other things, as we have already gone in prod.
